### PR TITLE
build: verify tutorial cache directory exists

### PR DESCRIPTION
**Pull request by anantmittal**

### CODE REVIEW

File: resources/src/ssec_tutorials/setup.py [hunk 1]
```diff
@@ -28,14 +28,22 @@
...
+QDRANT_ZIP_FILE = os.environ.get("QDRANT_ZIP_FILE", "scipy_qdrant_100k.zip")
 QDRANT_FOLDER_NAME = os.environ.get("QDRANT_FOLDER_NAME", "scipy_qdrant")
 QDRANT_PATH = TUTORIAL_CACHE / QDRANT_FOLDER_NAME
 QDRANT_COLLECTION_NAME = "arxiv_astro-ph_abstracts_astropy_github_documentation"
 
 
+def _verify_tutorial_cache_exits():
+    # Check if TUTORIAL_CACHE exists
+    if not TUTORIAL_CACHE.exists():
+        TUTORIAL_CACHE.mkdir(parents=True, exist_ok=True)
```
Comment (lsetiawan) [inline resources/src/ssec_tutorials/setup.py:40]: This is not needed has the code above will make the directory and it's parents by default.

**Review by lsetiawan — CHANGES_REQUESTED** [review]
The verify doesn't really need to happen. I think it would be better to do an assert after the `TUTORIAL_CACHE.mkdir` call in the existing code lines above. What you have has too many duplications.

**Comment by lsetiawan** [comment]
@anantmittal can you confirm that this works now?

**Review by lsetiawan — APPROVED** [review]

**Comment by anantmittal** [comment]
> @anantmittal can you confirm that this works now?

Yes, this works.

**Pull request merged**